crates/traits/Cargo.toml

```diff
@@ -0,0 +1,58 @@
+[package]
+authors = ["Parallel Team"]
+edition = "2021"
+name = "pallet-traits"
```

Comment:
I've been thinking about moving the contents of this pallet into other pallets and remove `pallet-traits`, but since both the `currency` and `loans` pallets depend on it, I'm not sure it's possible.

Reply:
It might be nice to merge this pallet yea, what are the dependencies exactly?

Reply:
The main traits `pallet_traits` exports are [LoansApi](https://github.com/interlay/interbtc/blob/ca6eccfa75ab92e27d6bd63e193ec8d2db085405/crates/traits/src/loans.rs#L22) and [OracleApi](https://github.com/interlay/interbtc/blob/ca6eccfa75ab92e27d6bd63e193ec8d2db085405/crates/traits/src/lib.rs#L25). 

The `currency` pallet uses both traits to define the default `CurrencyConversion` trait implementation, so it depends on `pallet_traits`. The `oracle` pallet depends on `currency`. If I moved `OracleApi` to the `oracle` pallet then I'd get a circular dependency between `currency` and `oracle`. 

The other solution would be to move the entire contents of `pallet_traits` to the `currency` pallet but this would add some responsibility creep to `currency`.